.github/workflows/foreman_plugin_js.yml

```diff
@@ -121,6 +121,11 @@ jobs:
           path: |
             foreman/package-lock.json
             ${{ format('{0}/package-lock.json', inputs.plugin) }}
+      - name: Archive all_react_app_exports
```

Comment:
I don't think we run `webpack:compile` in the plain JS tests, so this file will never be there?

Reply:
![image](https://github.com/theforeman/actions/assets/94284/af335454-7169-44e1-8c28-cfe1a8faf3e6)

Reply:
removed